Replace win.clipboard with gio clipboard api

beansgum opened the pull request:
This pr replaces the default `win.clipboard`  with gio's clipboard which can be accessed from any page rather than requiring access to `app.Window`.

Closes #409

itswisdomagain commented on README.md:
```diff
@@ -42,53 +42,60 @@ git clone https://github.com/raedahgroup/godcr %GOPATH%/src/github.com/raedahgro
 **Note: Building on Windows**
-Exporting `GO111MODULE` directly in CLI does not work and it is recommended to trigger
-via a `.bat` file.
+Exporting `GO111MODULE` directly in CLI does not work and
```
Don't know who added this to the readme, doesn't sound very accurate. Will need someone running Windows OS to confirm this.

barisere replied:
I've seen that elsewhere too. I think it's because of the difference in handling of environment variables between *nix and Windows.

itswisdomagain commented:
@barisere the 3 issues you identified above are all related to the interfaces not being able to find the required assets. Related issue https://github.com/raedahgroup/godcr/issues/239.

Sirmorrison commented on cmd/godcr-cli/log.go:
```diff
@@ -0,0 +1,175 @@
+	logDir, _ := filepath.Split(logFile)
+	err := os.MkdirAll(logDir, 0700)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "failed to create log directory: %v\n", err)
```
`Failed`

Sirmorrison commented on cmd/godcr-cli/log.go:
```diff
@@ -0,0 +1,175 @@
+	}
+	r, err := rotator.New(logFile, 10*1024, false, 0)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "failed to create file rotator: %v\n", err)
```
`Failed`

Sirmorrison commented on cmd/godcr-fyne/fyne.go:
```diff
@@ -0,0 +1,152 @@
+
+	// check if user passed commands/options/args in non-cli (fyne) mode
+	if len(args) > 0 {
+		fmt.Fprintf(os.Stderr, "unexpected command or flag in %s mode: %s\n",
```
`Unexpected`

Sirmorrison commented on cmd/godcr-nuklear/log.go:
```diff
@@ -0,0 +1,175 @@
+	logDir, _ := filepath.Split(logFile)
+	err := os.MkdirAll(logDir, 0700)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "failed to create log directory: %v\n", err)
```
`Failed`

Sirmorrison commented on cmd/godcr-nuklear/log.go:
```diff
@@ -0,0 +1,175 @@
+	}
+	r, err := rotator.New(logFile, 10*1024, false, 0)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "failed to create file rotator: %v\n", err)
```
`Failed`

Sirmorrison commented on cmd/godcr-terminal/log.go:
```diff
@@ -0,0 +1,175 @@
+	logDir, _ := filepath.Split(logFile)
+	err := os.MkdirAll(logDir, 0700)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "failed to create log directory: %v\n", err)
```
`Failed`

Sirmorrison commented on cmd/godcr-terminal/log.go:
```diff
@@ -0,0 +1,175 @@
+	}
+	r, err := rotator.New(logFile, 10*1024, false, 0)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "failed to create file rotator: %v\n", err)
```
`Failed`

Sirmorrison commented on cmd/godcr-terminal/terminal.go:
```diff
@@ -0,0 +1,155 @@
+
+	// check if user passed commands/options/args in non-cli (terminal) mode
+	if len(args) > 0 {
+		fmt.Fprintf(os.Stderr, "unexpected command or flag in %s mode: %s\n",
```
`Unexpected`

Sirmorrison commented on cmd/godcr-web/log.go:
```diff
@@ -0,0 +1,175 @@
+	logDir, _ := filepath.Split(logFile)
+	err := os.MkdirAll(logDir, 0700)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "failed to create log directory: %v\n", err)
```
same as above.

Sirmorrison commented:
<img width="426" alt="image" src="https://user-images.githubusercontent.com/27733432/64236166-bcb61300-cef1-11e9-80f2-d0fbea58dcd8.png">

unused import error on cli.

Sirmorrison commented:
PR looks good for fyne interface.

beansgum committed:
Replace win.clipboard with gio clipboard api

This pr replaces the default win.clipboard  with gio's clipboard which
can be accessed from any page rather than requiring access to app.Window.
